src/main/webapp/WEB-INF/views/cars/all.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All Cars</title>
+</head>
+<body>
+<h1>All Cars Page</h1>
+
+<table border="1">
+    <tr>
+        <th>ID</th>
+        <th>Model</th>
+        <th>Manufacturer</th>
+        <th>Drivers</th>
+    </tr>
+    <c:forEach var="car" items="${cars}">
+        <tr>
+            <td>
+                <c:out value="${car.id}"/>
+            </td>
+            <td>
+                <c:out value="${car.model}"/>
+            </td>
+            <td>
+                <c:out value="${car.manufacturer}"/>
+            </td>
+            <td>
+                <c:out value="${car.drivers}"/>
+            </td>
+            <td>
+                <a href="${pageContext.request.contextPath}/cars/delete?id=${car.id}">Delete</a>
+            </td>
+        </tr>
+    </c:forEach>
+</table>
+</body>
+</html>
```

Comment:
need to add empty line in the end of all classes and files

Reply:
![emptyline](https://user-images.githubusercontent.com/75136970/105239564-85dbae00-5b75-11eb-91d7-6d0e5a479a79.png)

Reply:
I have them in every file) Checked very carefully